clab/config.go

```diff
@@ -161,6 +172,9 @@ func (c *CLab) createNodeCfg(nodeName string, nodeDef *types.NodeDefinition, idx
 		Publish:         c.Config.Topology.GetNodePublish(nodeName),
 		Sysctls:         make(map[string]string),
 		Endpoints:       make([]*types.Endpoint, 0),
+		Sandbox:         nodeDef.Sandbox,
+		Kernel:          nodeDef.Kernel,
+		Runtime:         nodeDef.Runtime,
```

Comment:
I think these 3 items need functions like `c.Config.Topology.GetNodeSandbox()`, `c.Config.Topology.GetNodeKernel()`, and `c.Config.Topology.GetNodeRuntime()`.

To be able to have a node inherit the values from the `kinds` and `defaults` sections.

Reply:
ok, yep, makes sense

Reply:
I saw you added `nodeDef.GetSandbox()`,... but not the get methods from `Topology`

Reply:
oh, sry, I misunderstood. do you mean something like this?
```
func (t *Topology) GetNodeSandbox(name string) string {
	if ndef, ok := t.Nodes[name]; ok {
		if ndef.GetNodeSandbox() != "" {
			return ndef.GetNodeSandbox()
		}
		if t.GetKind(t.GetNodeKind(name)).GetNodeSandbox() != "" {
			return t.GetKind(t.GetNodeKind(name)).GetNodeSandbox()
		}
		return t.GetDefaults().GetNodeSandbox()
	}
	return ""
}
```

Reply:
yes, exactly